diagnostics-app/src/app/telemetry/telemetry-dashboard/telemetry-dashboard.component.html

```diff
@@ -1,17 +1,11 @@
 <!--
- Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
Hmmm I think it's better to retain earliest copyright year only.

Reply:
Done.